trinity/utils/async_iter.py

```diff
@@ -0,0 +1,21 @@
+from typing import (
+    AsyncIterable,
+    Set,
+)
+
+
+async def contains_all(async_gen: AsyncIterable[str], keywords: Set[str]) -> bool:
```

Comment:
This appears to be duplicated both here in `trinity.utils.async_iter` and under `trinity.tools.utils.async_iter`